bedrock/firefox/templates/firefox/profile/profile-base.html

```diff
@@ -0,0 +1,82 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% from "macros.html" import fxa_email_form with context %}
+
+{% add_lang_files "firefox/profile-per-install" %}
+
+{% extends "firefox/base-protocol.html" %}
+
+{% block page_title %}{{ _('An Important Change in Firefox: Dedicated Profiles') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('profile_per_install') }}
+{% endblock %}
+
+{% block body_class %}state-fxa-default{% endblock %}
+
+{% block site_header %}
+<header id="page-header" class="page-header">
```

Comment:
We should also think about turning this into a Protocol component perhaps? This kind of  header features on a few different pages now, such as here and `/whatsnew/`, `/firefox/new/` etc?

Reply:
Yeah I think it probably should. I already swiped it from whatsnew for this page so it's a perfect example of reusable code.